astroid/nodes/scoped_nodes.py

```diff
@@ -65,13 +66,19 @@
     InconsistentMroError,
     InferenceError,
     MroError,
+    StatementMissing,
     TooManyLevelsError,
 )
 from astroid.interpreter.dunder_lookup import lookup
 from astroid.interpreter.objectmodel import ClassModel, FunctionModel, ModuleModel
 from astroid.manager import AstroidManager
 from astroid.nodes import Arguments, Const, node_classes
 
+if sys.version_info >= (3, 6, 2):
```

Comment:
I just realized that we also need to do https://github.com/PyCQA/pylint/issues/5065 for astroid.

Reply:
I believe this is the only one. Shall I open an issue so we don't forget about this?

Reply:
Might as well create an issue from the [unofficial todo list](https://github.com/PyCQA/pylint/issues/5065#issuecomment-935428793) (hidden in dozen of comment 😄) and add a "release astroid 2.8.5 for > 3.6.2 only" ?: